libraries/AP_HAL_ChibiOS/hwdef/MatekH743-periph/hwdef-bl.dat

```diff
@@ -0,0 +1,4 @@
+# hw definition file for processing by chibios_pins.py
+# for Matek H743-periph bootloader
+
+include ../MatekH743/hwdef-bl.dat
```

Comment:
This bootloader needs:
```
PD0 CAN1_RX CAN1
PD1 CAN1_TX CAN1

# reserve 256 bytes for comms between app and bootloader
RAM_RESERVE_START 256

# use DNA
define HAL_CAN_DEFAULT_NODE_ID 0
define CAN_APP_NODE_NAME "org.ardupilot.MatekH743-periph"
```

Reply:
valid point.

Reply:
Begs a question about board ID if we're requiring a different bootloader.

Reply:
we should really just add CAN to all of our normal bootloaders that it can fit in

Reply:
I've added it to the periph bootloader in this PR - will chat to @tridge about putting those changes in the parent though, as I think it would be better

Reply:
I think add CAN only in child bootloader. I think having CAN by default for bl on this board doesn't make much sensor, as USB is so easy on it

Reply:
@tridge the point would be to have a single bootloader to rule them all